lib/project-info-manager.js

```diff
@@ -217,8 +217,8 @@ const projectInfoTemplate = {
 	"ProjectPath": "",
 	"SourceDir": "src",
 	"DistributionDir": "build",
-	"BuildCommand": "npm run-script build",
-	"StartCommand": "npm run-script start",
+	"BuildCommand": (/^win/.test(process.platform) ? "npm.cmd run-script build" : "npm run-script build"),
```

Comment:
This is configurated on during init (we can add later a default script for windows) this should be discarded

Reply:
I edited these lines because most windows users do not know that they have to use `npm.cmd` instead of `npm` when initialise a project with `awsmobile-cli`. I think my changes can help windows users to choose the correct default values.